app/components/SCForm.js

```diff
@@ -112,16 +115,18 @@ class SCForm extends Component {
       if (fieldValue !== undefined) {
         if (type === 'number') {
           fieldValue = +[fieldValue];
-          ruleRunner(field.fieldKey, field.fieldLabel,
+          const numRuleRunner = ruleRunner(field.field_key, field.field_label,
                 ErrorMessages.mustBeANum(field.type, field.fieldValue));
           fieldValidations.push(ruleRunner);
         } else if (field.type === 'string') {
-          ruleRunner(field.fieldKey, field.fieldLabel,
-               ErrorMessages.strValidation(min, max, fieldValue));
+          const strMax = ruleRunner(field.field_key, field.field_label,
+              ErrorMessages.strMax(max, fieldValue));
+          const strMin = ruleRunner(field.field_key, field.field_label,
+              ErrorMessages.strMin(min, fieldValue));
           fieldValidations.push(ruleRunner);
```

Comment:
Also, you only want one `ruleRunner` for each field. So it should be something like `ruleRunner(key, label, strMin, strMax, required, ...etc)`